test/lib/autoinst_issues/invalid_value_test.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative '../../test_helper.rb'
+require 'autoinstall/autoinst_issues/invalid_value'
+
+describe Y2Autoinstallation::AutoinstIssues::InvalidValue do
+  subject(:issue) do
+    described_class.new('firewall', 'interfaces', 'eth0',
+                        'This interface has been defined for more than one zone.', :fatal)
+  end
+
+  describe '#message' do
+    it 'includes relevant information' do
+      message = issue.message
+      expect(message).to include 'interfaces'
+      expect(message).to include 'eth0'
+    end
+  end
+
+  describe '#severity' do
+    it 'returns :fatal' do
```

Comment:
Well, it only returns `:fatal` when it is set to `:fatal`. You might want to rephrase it (or to implement two different cases).